flags.go

```diff
@@ -31,6 +31,12 @@ const (
 	nilValue   = "<nil>"
 )
 
+var (
+	varnameMap = map[string]string{
+		"stringToString": "string",
```

Comment:
to run by docs but its there a better description to highlight the value must be `key=value`

Reply:
I've checked with @sarahsimpers from docs and she told me it should be `string`.
Not sure what they think about the `key=value` proposal. I'll ping them.

Reply:
I think this should be `string`. This is displaying under "type" in the docs, and usually we would only show a datatype there. The _format_ would be `key=value`, but we could tell them that in the description and show it in the example commands because the type is still string unless I'm misunderstanding.

Reply:
Addressed in #52